### PR TITLE
Pretty up the landing page.

### PR DESCRIPTION
**Pull request by chuckharmston**
r? @ngokevin 

The content in the UX doc ([page 10](https://drive.google.com/a/mozilla.com/file/d/0By5ZvBZQRgjRNW9Wek1HT3BDb1U/view)) was all FPO, so...everything is terrible. I'm asking around for somebody to write some real content. For now:

# Signed Out

screen shot 2015-09-28 at 2 04 28 pm

# Signed In

screen shot 2015-09-28 at 2 04 38 pm

### CODE REVIEW

File: src/media/js/site/containers/landing.js [hunk 1]
```diff
@@ -1,17 +1,65 @@
 import React from 'react';
+import {connect} from 'react-redux';
 import {ReverseLink} from 'react-router-reverse';
+import {history} from 'react-router/lib/BrowserHistory';
+import {bindActionCreators} from 'redux';
 
-import {Page} from '../../site/components/page';
+import {fxaLoginBegin, login} from '../actions/login';
```
Comment (ngokevin) [inline src/media/js/site/containers/landing.js]: we can get rid of this whitespace

**Comment by pwalm** [comment]
Nice! Love it. :D

File: src/media/js/site/containers/landing.js [hunk 2]
```diff
@@ -1,17 +1,65 @@
...
 
+import {LoginButton} from '../components/login';
+import {Page} from '../components/page';
+
+
+class Landing extends React.Component {
```
Comment (ngokevin) [inline src/media/js/site/containers/landing.js]: proptypes

File: src/media/js/site/containers/landing.js [hunk 3]
```diff
@@ -1,17 +1,65 @@
...
 
-export default class Landing extends React.Component {
   render() {
     return (
-      <Page title="Develop HTML5 content for an open marketplace.">
-        <ul>
-          <li><ReverseLink to="addon">Firefox OS Add-ons</ReverseLink></li>
-        </ul>
+      <Page className="landing">
+        <header className="landing--header">
```
Comment (ngokevin) [inline src/media/js/site/containers/landing.js]: i'd just use a div, we have a header element already and this doesn't really look like a header

File: src/media/js/site/containers/landing.js [hunk 4]
```diff
@@ -1,17 +1,65 @@
...
+          <h1>New for Firefox OS&hellip;Add-ons.</h1>
+          <h2>Just like for the browser. Only for mobile.</h2>
+          <p>
+            <a href="https://developer.mozilla.org/en-US/docs/Mozilla/Firefox_OS/Add-ons"
```
Comment (ngokevin) [inline src/media/js/site/containers/landing.js]: en-US

File: src/media/js/site/containers/landing.js [hunk 5]
```diff
@@ -1,17 +1,65 @@
 import React from 'react';
```
Comment (ngokevin) [inline src/media/js/site/containers/landing.js]: congrats, you've been randomly selected to write a test for this container

File: src/media/js/site/containers/landing.js [hunk 6]
```diff
@@ -1,17 +1,65 @@
...
+import {connect} from 'react-redux';
 import {ReverseLink} from 'react-router-reverse';
+import {history} from 'react-router/lib/BrowserHistory';
```
Comment (ngokevin) [inline src/media/js/site/containers/landing.js]: no need

File: src/media/js/site/containers/landing.js [hunk 7]
```diff
@@ -1,17 +1,72 @@
...
   render() {
     return (
-      <Page title="Develop HTML5 content for an open marketplace.">
-        <ul>
-          <li><ReverseLink to="addon">Firefox OS Add-ons</ReverseLink></li>
-        </ul>
+      <Page className="landing">
+        <header className="landing--header">
+          <h1>New for Firefox OS&hellip;Add-ons.</h1>
+          <h2>Just like for the browser. Only for mobile.</h2>
```
Comment (ngokevin) [inline src/media/js/site/containers/landing.js]: alternative copy:

```Everything changes...again```

or

```This is our moment...in time```

File: src/media/js/site/containers/landing.js [hunk 8]
```diff
@@ -1,17 +1,72 @@
...
+               target="_blank">
+              Learn how to make a Firefox OS add-on &rsaquo;
+            </a>
+          </p>
+        </header>
+        <div className="landing--content">
+          <section className="landing--submit">
+            <p>Submit a new add-on or manage your current add-ons:</p>
+            <ReverseLink className="button" to="addon-dashboard">
+              Submit a Firefox OS Add-On &rsaquo;
```
Comment (ngokevin) [inline src/media/js/site/containers/landing.js]: the copy says Submit but it links to addon dashboard

File: src/media/js/site/containers/landing.js [hunk 9]
```diff
@@ -1,17 +1,72 @@
...
+import {bindActionCreators} from 'redux';
 
-import {Page} from '../../site/components/page';
+import {fxaLoginBegin, login} from '../actions/login';
+import {LoginButton} from '../components/login';
+import {Page} from '../components/page';
 
 
-export default class Landing extends React.Component {
+class Landing extends React.Component {
```
Comment (ngokevin) [inline src/media/js/site/containers/landing.js]: you'll want to export it for tests

**Comment by ngokevin** [comment]
rwc

File: src/media/js/site/containers/landing.js (hunk 8 above)
Comment (chuckharmston) [inline src/media/js/site/containers/landing.js]: Yep, and the copy above it says "submit or manage". ¯\_(ツ)_/¯

Comment (ngokevin) [inline src/media/js/site/containers/landing.js]: and it places submit first

File: src/media/js/site/containers/__tests__/landing.test.js [hunk 10]
```diff
@@ -0,0 +1,26 @@
+import {Landing} from '../landing';
+
+
+describe('Landing', () => {
+  jsdom();
+
+  const props = {
+    children: {},
```
Comment (ngokevin) [inline src/media/js/site/containers/__tests__/landing.test.js]: don't need this

File: src/media/js/site/containers/__tests__/landing.test.js [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+    fxaLoginBegin: () => {},
+    login: () => {},
+    user: {},
+    signTOS: () => {},
+    siteConfig: {},
+  };
+
+  describe('Landing', () => {
```
Comment (ngokevin) [inline src/media/js/site/containers/__tests__/landing.test.js]: you have two duplicate ```describe('Landing')```s and trying to set up jsdom twice

**Pull request merged**